app/lib/features/activities/widgets/invitation_card.dart

```diff
@@ -80,17 +100,14 @@ class InvitationCard extends ConsumerWidget {
       subtitle: Wrap(
         children: [
           Text(L10n.of(context).invitationToSpace),
-          inviter(
-            context,
-            ref,
-          ),
+          inviter(context),
         ],
       ),
     );
   }
 
-  ListTile renderGroupChatTile(BuildContext context, WidgetRef ref) {
-    final roomId = invitation.roomIdStr();
+  ListTile renderGroupChatTile(BuildContext context) {
+    final roomId = widget.invitation.roomIdStr();
```

Comment:
`invitation.room().displayName();` isn't working here?

Reply:
nope, it's a future